File: android/src/main/kotlin/com/syncrhrohealth/bandpass_ble_connection/core/command/ResetFuelLevelSetCmd.kt
package com.syncrhrohealth.bandpass_ble_connection.core.command

import com.syncrhrohealth.bandpass_ble_connection.core.command.builder.CommandBuilder
import com.syncrhrohealth.bandpass_ble_connection.core.enum.CommandCode
import com.syncrhrohealth.bandpass_ble_connection.core.handler.DeviceHandler


object ResetFuelLevelSetCmd {
    fun send(handler: DeviceHandler) {
        val builder = CommandBuilder.build(CommandCode.PACKET_CMD_RESET_FUEL_LEVEL_SET)

        handler.write(builder)
    }

}

File: ios/Classes/Core/Command/ResetFuelLevelSetCmd.swift
//
//  ResetFuelLevelSetCmd.swift
//  Runner
//
//  Created by MAC on 11/3/25.
//

import Foundation

struct ResetFuelLevelSetCmd {
    static func send(handler: DeviceHandler) {
        let builder = CommandBuilder.build(commandCode: .packetCmdResetFuelLevelSet)
        handler.write(data: builder)
    }
}
